tests/unit/includes/backends/class-omise-backend-installment-test.php

```diff
@@ -247,29 +247,3 @@ public function correctly_calculating_monthly_payment_amount_as_merchant_absorbs
 		$this->assertEquals(833.33, $result);
 	}
 }
-
-/**
- * Mock Omise_Capabilities class.
- * NOTE: This might not be an ideal way to mock a class,
- *       feel free to enhance the test or the core code.
- *
- * @see includes/class-omise-capabilities
- */
-class Omise_Capabilities
```

Comment:
Removing this because it is no use and we cannot have 2 same class (this and the actual class which included in Omise_Capabilities test).